### ISSUE TITLE
Cannot use import statement outside a module

### ISSUE THREAD

**Issue by epleaner** [ueno-llc/react-three-gui#6]
Getting this when trying to use react-three-gui with webpack. 

node_modules/react-spring/three.js:1
import * as THREE from 'three';
^^^^^^

SyntaxError: Cannot use import statement outside a module

**Comment by talentlessguy** [comment]
This is not a `react-three-gui` issue. it is `three` issue, because the module system is broken there. I don't know if you use server rendering or not, but all the code must be run from client-side.

**Comment by epleaner** [comment]
Thanks for responding – when you say "all the code", do you mean
react-three-gui? Or anything three-related?
- Eli Pleaner

**Comment by talentlessguy** [comment]
@epleaner yes, everything three related is most likely to break on server side due to not having CJS interop (afaik)

also, it depends on your webpack config

I experienced similar problems when used Next.js. The solution was transpiling modules or using client side only
